ets-demo/CHANGES.txt

```diff
@@ -0,0 +1,17 @@
+ETS Demo Changelog
+==================
+
+Release 0.1.0
+-------------
+
+This is an initial release of the Enthought Tool Suite Demo GUI application.
+The package itself does not include any demo materials. It relies on other
+projects to contribute data files via entry points.
+
+The application was initially hosted and advertised via
```

Comment:
🇺🇸 advertized

Reply:
Ah actually, "advertised" is more popular. https://books.google.com/ngrams/graph?content=advertize%2Cadvertise&year_start=1800&year_end=2000&corpus=28&smoothing=3